### DISCUSSION TITLE
Group of Subscribers

### DISCUSSION THREAD

**Discussion by Natha** [buttondown/discussions#11] [Q&A] [answered]
Hey there,

I was wondering if I was the only one missing some kind of group functions to order my subscribers 🤔

Right now, Buttondown only allows to filter subscriber either by subscribing status or UTM Sources :

<img width="307" alt="Screenshot 2023-04-11 at 00 10 25" src="https://user-images.githubusercontent.com/241754/231008585-be4a2286-63d5-4268-ad0b-86e599f1302a.png">

Me and my team are working on a bunch of multi-lingual (FR/EN/NL) newsletters. So, to have a way to order/tag subscribers based on a language would probably be perfect. 

Is there a way to do that at the moment ? If no, I would love to add this as a feature request 😃

**Comment by jmduke** [comment, marked as answer]
Hello! I suspect you might want [tags](https://docs.buttondown.email/advanced-features/tags-vs-newsletters). When you create them, they'll show up in that filtering widget!

**Reply by Natha** [reply]
Yup, I was about to delete my post after figuring out my mistake. Thanks :)

**Reply by jmduke** [reply]
Ha, np!

**Comment by growdigital** [comment]
> Yup, I was about to delete my post after figuring out my mistake. Thanks :)

Glad you didn't, because I have exactly the same question!
